pom.xml

```diff
@@ -31,9 +31,9 @@
     <version.assertj>2.6.0</version.assertj>
 
     <!-- Overridden from parent -->
-    <jdk.min.version>1.5</jdk.min.version>
-    <maven.compiler.target>1.5</maven.compiler.target>
-    <maven.compiler.source>1.5</maven.compiler.source>
+    <jdk.min.version>1.8</jdk.min.version>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
What's the reason for updating to JDK8?

Reply:
I just reverted this, there was no technical reason for it.